openslides/agenda/static/js/agenda/agenda.js

```diff
@@ -153,6 +153,50 @@ angular.module('OpenSlidesApp.agenda', [])
       };
 })
 
-.controller('AgendaCSVImportCtrl', function($scope, Agenda) {
-    // TODO
+.controller('AgendaImportCtrl', function($scope, $state, Agenda) {
+    // import from textarea
+    $scope.importByLine = function () {
+        $scope.items = $scope.itemlist[0].split("\n");
+        $scope.importcounter = 0;
+        $scope.items.forEach(function(title) {
+            var item = {title: title};
+            item.weight = 0;  // TODO: the rest_api should do this
+            item.tags = [];   // TODO: the rest_api should do this
+            // TODO: create all items in bulk mode
+            Agenda.create(item).then(
+                function(success) {
+                    $scope.importcounter++;
+                }
+            );
+        });
+    }
```

Comment:
One possible way would be, that Agenda.create() returned the id of the new created element. Then the ids could be saved in an array. If at least one create-request fails, the user could be ask if he want to delete all the elements, which were created successfully.